locales/en.json

```diff
@@ -1,7 +1,29 @@
 {
+  "ErrorIndividualPackagesUnsupported": "Error: In manifest mode, `vcpkg install` does not support individual package arguments.\nTo install additional packages, edit vcpkg.json and then run `vcpkg install` without any package arguments.",
+  "ErrorInvalidClassicModeOption": "Error: The option {value} is not supported in classic mode and no manifest was found.",
+  "ErrorInvalidManifestModeOption": "Error: The option {value} is not supported in manifest mode.",
+  "ErrorNoVSInstance": "Error: in triplet {triplet}: Unable to find a valid Visual Studio instance",
+  "ErrorNoVSInstanceAt": "     at \"{path}\"",
+  "_ErrorNoVSInstanceAt.comment": "Printed after ErrorNoVSInstance on a separate line",
+  "ErrorNoVSInstanceFullVersion": "    with toolset version prefix {version}",
+  "_ErrorNoVSInstanceFullVersion.comment": "Printed after ErrorNoVSInstance on a separate line",
+  "ErrorNoVSInstanceVersion": "    with toolset version {version}",
+  "_ErrorNoVSInstanceVersion.comment": "Printed after ErrorNoVSInstance on a separate line",
+  "ErrorRequireBaseline": "Error: this vcpkg instance requires a manifest with a specified baseline in order to interact with ports. Please add 'builtin-baseline' to the manifest or add a 'vcpkg-configuration.json' that redefines the default registry.\n",
```

Comment:
Similarly, I think we should always have the `\n` or not, with a mild preference for no `\n` in the localized text from me.

Reply:
Yeah, this was a bug.